README.md

```diff
@@ -51,93 +39,8 @@ To setup your development environment, please check [this guide](docs/CONTRIBUTI
 
 # Features
 
-You can watch a short video with a tour of some of the features of the Galaxy Tools extension here:
+For a complete list of features [check this section](client/README.md#features).
 
-[![Galaxy Tools features video](https://img.youtube.com/vi/MpPrgtNrEcQ/0.jpg)](https://www.youtube.com/watch?v=MpPrgtNrEcQ)
-
-## Tag and attribute auto-completion
-
-![Demo feature auto-completion](../assets/feature.autocompletion.gif)
-
-The tags and attributes are suggested based on the [Galaxy.xsd](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/xsd/galaxy.xsd) schema. They will appear in the same order as they are declared in the schema to comply with the best practices recommendations defined in the [Galaxy IUC Standards Style Guide](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?#coding-style).
-
-## Documentation on Hover
-
-![Demo feature hover documentation](../assets/feature.hover.documentation.gif)
-
-The documentation of tags and attributes is retrieved from the [Galaxy.xsd](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/xsd/galaxy.xsd) schema.
-
-> Please note that some elements in the schema are still missing documentation. This will probably be improved over time.
-
-## Document validation
-
-![Demo feature validation](../assets/feature.validation.png)
-
-The tools are also validated against the [Galaxy.xsd](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tool_util/xsd/galaxy.xsd) schema.
-
-## Document auto-formatting
-
-![Demo feature auto-formatting](../assets/feature.autoformat.gif)
-
-When the tool file is saved it gets auto-formatted to maintain a consistent format with the [Galaxy IUC Standards Style Guide](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?#coding-style).
-
-## Tag auto-closing
-
-![Demo feature auto-close tags](../assets/autoCloseTag.gif)
-
-Whenever you write a closing (`>`), the corresponding closing tag will be inserted. You can also type `/` in an open tag to close it.
-
-## Snippets
-
-![Demo snippets](../assets/snippets.gif)
-
-Snippets can be really helpful to speed up your tool wrapper development. They allow to quickly create common blocks and let you enter just the important information by pressing `tab` and navigating to the next available value.
-
-> If you want to add more snippets check the [guide](./docs/CONTRIBUTING.md#adding-snippets) in the contribution guidelines.
-
-## Embedded syntax highlighting
+You can also watch a (somewhat old) short video with a tour of some of the features of the Galaxy Tools extension here:
 
-![Demo feature embedded syntax highlighting](../assets/feature.embedded.syntax.png)
-
-Basic support for `Cheetah` and `reStructuredText` syntax highlighting inside the `<command>`, `<configfile>` and `<help>` tags. The embedded code should be inside a `CDATA` block.
-
-## Auto-generate tests
```

Comment:
why is this all removed?

Reply:
Instead of duplicating the features list in the **main** readme and in the **client** readme, [the main read now references the features section in the client readme](https://github.com/galaxyproject/galaxy-language-server/blob/master/README.md#features)

Reply:
Ok, but not all of your cool videos are linked, isn't it. Just don't want to loose them :)

Reply:
But... now the images are not displayed in Github because of the "trick" of the `assets` branch, so I guess I should revert that change :man_shrugging: